### ISSUE TITLE
Is it possible to bind fn2 and fn3 to keys? (GK68XS)

### ISSUE THREAD

**Issue by mplusp** [pixeltris/GK6X#83]
Hey there, is it somehow possible to have fn2 and fn3 keys as it is actually advertised on the [GK68XS' Indiegogo page](https://www.indiegogo.com/projects/epomaker-gk68xs-bluetooth-mechanical-keyboard#/)?

I would want fn2/fn3 to act just like the fn key in terms of functionality (but mapped to other keys/macros, obviously) but couldn't figure out how to actually configure this through the official software under Windows. Then I found this great Tool, but it seems as if it's not possible using this, either.

It would be great, if we could somehow change any key to fn2/fn3, but if it's only possible using the split spacebar, that would be good, too.

Thanks for your time :)

**Comment by pixeltris** [comment]
The way those split spacebar keys work is hard coded into the firmware. They are labelled as 'fn' keys but aren't actually fn keys. They are just keys which can be mapped when held down, but also behaves like a spacebar key when tapped.

The way it's designed to work is that you're expected to map those extra spacebar keys to one of the temporary layer switch keys, which effectively gives you something similar to the behavior of fn.

**Comment by mplusp** [comment]
Oh, that's what I feared. The thing is, that I use the layers 1 and 2 as base layers for my Windows and Mac machines, because the standard layer isn't changeable, at least not in the standard software. That's why those two layers aren't available for the temp layer switching.

I saw, that it should be able to change the standard layer with this tool. Maybe I'll get it figured out this way.

Thanks for the help :)
